.github/workflows/packaging.yml

```diff
@@ -3,6 +3,14 @@ name: packaging
 on: [push, pull_request]
 
 jobs:
+  version-check:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Run action
```

Comment:
Let's add the same

```
    # Skip pull request jobs when the source branch is in the same
    # repository.
    if: |
      github.event_name == 'push' ||
      github.event.pull_request.head.repo.full_name != github.repository
```

skip so there won't a double job execution for each PR push

Reply:
Done, added